blocks/columns/columns.css

```diff
@@ -317,7 +332,7 @@ div.img-col > video {
   width: 100%;
 }
 
-.columns > div > div:not(.columns-img-col) div:not(.img-col) {
+.columns:not(.flex-start) > div > div:not(.columns-img-col) div:not(.img-col) {
```

Comment:
There's a `no-padding` variant to columns (if it's not in the sidekick library it might be my fault, sorry), maybe use that?

Reply:
Thank you @dnbute, i have used no-padding variation and did some minor changes around it. Please see the PR